src/NLog/Targets/FileTarget.cs

```diff
@@ -201,8 +200,17 @@ public FileTarget()
             this.previousFileNames = new Queue<string>(this.maxLogFilenames);
             this.fileAppenderCache = FileAppenderCache.Empty;
             this.CleanupFileName = true;
+            
+            this.Compressor = DefaultCompressor;
```

Comment:
Compressor is per FileTarget Instance and DefaultCompressor it a static property. Background is that you may wanna set a standard compression implementation for all FileTargets but may choose to set onother for a single instance. Maybe a bit too flexible ;)